src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -0,0 +1,199 @@
+			}
+
+		}
+
+		public void Show(ToastNotification notification)
+		{
+			var androidNotification = new Android.App.Notification();
+			androidNotification.Category = Android.App.Notification.CategoryMessage;
+
+			Android.App.Notification.Builder builder;
+
+			// if API < 11, then another version...
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.O)
+				builder = new Android.App.Notification.Builder(Android.App.Application.Context, _channelID);
+			else
+			{
+#pragma warning disable CS0618 // Type or member is obsolete
+				builder = new Android.App.Notification.Builder(Android.App.Application.Context);
+#pragma warning restore CS0618 // Type or member is obsolete
+			}
+
+
+			// extract <text> nodes from XML
+			// - much simpler than using XML parser (and, we don't have Windows.Data.Xml.Dom.XmlDocument implemented)
+
+			string toastBigText = "";
+			string toastText = "";
+			string xmlToast = notification._xmlDocument.GetXml();
+
+			int ind = xmlToast.IndexOf("<text>");
+			while (ind > 0)
+			{
+				xmlToast = xmlToast.Substring(ind + 6);
+				ind = xmlToast.IndexOf("<");
+				if (ind > -1)   // node can be empty...
+				{
+					if (toastBigText == "")
+					{
+						// first text - bigger text
+						toastBigText = xmlToast.Substring(0, ind);
+					}
+					else
+					{ // smaller text
+						if (toastText != "")
+							toastText += " ";   // "\n" doesn't work, but at least space should be added
+						toastText += xmlToast.Substring(0, ind);
+					}
+
+					ind = xmlToast.IndexOf("<text>"); // search for next text node
+				}
+			}
+
+			if (!string.IsNullOrEmpty(toastBigText))
+				builder.SetContentTitle(toastBigText);
+			if (!string.IsNullOrEmpty(toastText))
+				builder.SetContentText(toastText);
+
+
+			// we have to set icon ("This is the only user-visible content that's required", says doc).
+
+			// method 1
+			string packageName = Android.App.Application.Context.PackageName;
+			Android.Content.PM.ApplicationInfo info = null;
+			try
+			{
+				info = Android.App.Application.Context.ApplicationContext.PackageManager.GetApplicationInfo(packageName, 0);
+			}
+			catch
+			{
+			}
+
+			int iconId;
+			if (info != null)
+			{
+				iconId = info.Icon;
+			}
+			else
+			{
+				// try another method - using more resources (CPU and memory), as Intent is involved
+				Android.Content.Intent intent = Android.App.Application.Context.PackageManager.GetLaunchIntentForPackage(Android.App.Application.Context.PackageName);
+				Android.Content.PM.ResolveInfo resolveInfo = Android.App.Application.Context.PackageManager.ResolveActivity(intent, Android.Content.PM.PackageInfoFlags.MatchDefaultOnly);
+				iconId = resolveInfo.IconResource;
+			}
+
+			builder.SetSmallIcon(iconId);
+
+
+			// for somewhat older Android, we need to set Priority (deprecated in API26)
+
+			if (Android.OS.Build.VERSION.SdkInt < Android.OS.BuildVersionCodes.O)
+			{
+#pragma warning disable CS0618 // Type or member is obsolete

```

Comment:
What should be used instead ?

Reply:
Nothing. Priority is obsolete, more current is different channels for different priorities, and creating channel is in lines 21-25.